pom.xml

```diff
@@ -77,22 +77,22 @@
 
     <properties>
         <!-- Minimal Jenkins version (2.14.0 ) required by for pipeline job -->
-        <jenkins.version>1.642.3</jenkins.version>
+        <jenkins.version>2.289.3</jenkins.version>
         <!--<jenkins.version>2.14</jenkins.version>-->
         <!--<jenkins.version>2.60.3</jenkins.version>-->
-        <slf4j.version>1.7.5</slf4j.version>
+        <slf4j.version>1.7.30</slf4j.version>
         <java.level>8</java.level>
         <agent.version>2.9.5</agent.version>
         <skipTests>true</skipTests>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
     <dependencies>
+        
```

Comment:
Extra blank lines are usually fine, but we generally prefer to not have trailing white space.
```suggestion

```